### PR TITLE
Fix type issues in js files

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: plugins-bundle-analyzer/tsconfig.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "skipLibCheck": true,
```
Comment (oded-soffrin) [inline plugins-bundle-analyzer/tsconfig.json:11]: Like!

File: plugins-bundle-analyzer/bundlesSizesBaseline.json [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-  "wix-rich-content-plugin-hashtag": 232,
+  "wix-rich-content-plugin-file-upload": 213,
+  "wix-rich-content-plugin-gallery": 684,
+  "wix-rich-content-plugin-giphy": 211,
+  "wix-rich-content-plugin-hashtag": 214,
   "wix-rich-content-plugin-headers-markdown": 193,
   "wix-rich-content-plugin-headings": 194,
-  "wix-rich-content-plugin-html": 234,
-  "wix-rich-content-plugin-image": 234,
+  "wix-rich-content-plugin-html": 217,
```
Comment (oded-soffrin) [inline plugins-bundle-analyzer/bundlesSizesBaseline.json:12]: LIKE!

File: tsconfig.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "strictNullChecks": true,
     "esModuleInterop": true,
     "resolveJsonModule": true,
+    "downlevelIteration": true,
```
Comment (oded-soffrin) [inline tsconfig.json:17]: what was the trigger for adding it?

Comment (mharsat) [reply tsconfig.json:17]: Basically i got some Symbol.iterator error, can't remember where. I'll try and see if I can now build successfully without it

**Comment by oded-soffrin** [comment]
🥳

**Pull request merged**